### DISCUSSION TITLE
Automatic fit() time and samples size message

### DISCUSSION THREAD

**Discussion by michail-p** [microsoft/FLAML#311] [Ideas]
Hello, im working on a gesture recognition app that uses tabular data. As time passes im adding more gestures and its hard having to trial-and-error the balance between training time and samples per label.

It would be very helpful if the system decided how 
much time to run automatically and show a message to increase samples when they are not enough

**Comment by sonichi** [comment]
I've created a feature request for [how much time to run](https://github.com/microsoft/FLAML/wiki/Roadmap-for-Upcoming-Features#how-to-decide-value-of-time_budget).
I've also created a feature request for [deciding how many labeled training examples are needed](https://github.com/microsoft/FLAML/wiki/Roadmap-for-Upcoming-Features#decide-how-many-labeled-training-examples-are-needed). But for the specific function you are asking for, it's unclear what it means by "they are not enough". If you have a target of the accuracy, then you can check whether that target has been reached after fit(). I guess what you mean is to show the message even before the fit() finishes. In other words, you want to see an estimation of the best accuracy that can be expected from fit(). Is that right?

**Reply by michail-p** [reply]
You are right, i read both requests and made this thread as you suggested.

**Reply by sonichi** [reply]
#312 created.
